addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1253,6 +1253,44 @@ QUnit.test('load single message from channel initially', async function (assert)
     );
 });
 
+QUnit.test('Load a message from channel with an number as active_id', async function (assert) {
```

Comment:
Start with small case char (consistency with all other test names).

This test is not great in intent w.r.t. the change in PR.
It should test that channel correctly opens on load with providing active_id as number to discuss.

Assertions below should test that thread is correctly open, either from checking sidebar active thread or with `.o_Thread[data-thread-local-id]` (I prefer the later).